### DISCUSSION TITLE
Check for conditional job execution fails

### DISCUSSION THREAD

**Discussion by pantelis-karamolegkos** [community/community#36133] [Actions] [answered]
I have the following conditional check for a job execution

```
 if: inputs.push_to_prod && ( inputs.manual_version == "" || inputs.manual_build_branch != 'master' )
```

However the workflow validation fails as follows:

```
The workflow is not valid. .github/workflows/backend_image_build.yaml (Line: 41, Col: 13): Unexpected symbol: '""'. Located at position 51 within expression: inputs.push_to_prod && ( inputs.manual_version == "" || inputs.manual_build_branch != 'master' )
```

Why is that?

**Comment by ChristopherHX** [comment, marked as answer]
Double quotes`""` are invalid in actions expressions, you have to use single quotes `'`. I believe there was a feature request for double quotes, but nothing happened in expression syntax since 2019 ( the initial Release of Actions ).

This should work
```yaml
 if: inputs.push_to_prod && ( inputs.manual_version == '' || inputs.manual_build_branch != 'master' )
```
Weird is that you used the correct quotes for string `'master'`, but not for the empty string.
